src/renderer/windows/GPMWebView/interface/customNavigation/showCover.js

```diff
@@ -0,0 +1,3 @@
+Emitter.on('cover:show', () => {
+  document.querySelector('#hover-icon').click();
```

Comment:
I would prefer that this is done in a safer way, google change selectors all the time. Just assigning the DOM node to a variable and checking the variable isn't null before clicking would be a better thing to do 👍